spec/models/contact_spec.rb

```diff
@@ -1135,15 +1135,15 @@ def contacts_with_value_at(value,time)
       specify do
         @contact.history_entries.where(attribute: 'level').each{|h|[
             DateTime.civil(2012,11,26,18,0,0,0),
-            DateTime.civil(2012,11,26,18,58,21,0),
-            DateTime.civil(2012,11,26,23,41,16,0)
+            DateTime.civil(2012,11,27,18,58,21,0),
+            DateTime.civil(2012,11,28,23,41,16,0)
         ].should include h.changed_at }
       end
 
       example { contacts_with_value_at('sádhaka', 1.year.ago).should_not include @contact }
       example { contacts_with_value_at('sádhaka', DateTime.civil(2012,11,26,18,57,0,0)).should include @contact }
       example { contacts_with_value_at(nil,1.year.ago).should include @contact }
-      example { contacts_with_value_at('aspirante','2012-11-26 23:00:00 UTC').should include @contact }
+      example { contacts_with_value_at('aspirante','2012-11-26 23:00:00 UTC').should_not include @contact }
```

Comment:
Line is too long. [107/80]<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.